tests/relay/mptt/models.py

```diff
@@ -1,9 +1,132 @@
+import django
 from django.db import models
 from mptt.fields import TreeForeignKey
 from mptt.models import MPTTModel
+from mptt.models import MPTTModelBase as _MPTTModelBase
 
+if django.VERSION >= (5, 1):
```

Comment:
**question:** why is this needed?

Can you add a comment with the explanation, and also pointing out where this code came from?